### DISCUSSION TITLE
Does ignoring update eg. 0.0.1 to 0.0.2 will also block future upgrade to 0.0.3?

### DISCUSSION THREAD

**Discussion by gustaff-weldon** [renovatebot/renovate#12023] [Request Help] [answered]
As in tittle, I'm curious how does the closing/ignoring PR with update works.
While testing renovate bot integration we have closed a couple of renovate PRs. They show up now on renovate dashboard in igored. 

I wonder how this works, Eg. does ignoring update eg. from 0.0.1 to 0.0.2 will also block future PR with upgrade to 0.0.3?

According to dashboard docs https://docs.renovatebot.com/key-concepts/dashboard/#visibility-into-rejecteddeferred-updates one can always bring the update back, but it does not say what will happen to any future updates if I keep it in ignored.

**Comment by viceice** [comment]
With default configuration renovate will only ignore the specific version

**Comment by rarkins** [comment, marked as answer]
It doesn't block 0.0.3 and the PR comment you get in the closed PR should explicitly mention it's 0.0.2 ignored specifically. 

For major updates closing the PR will ignore major updates. Eg if you close an update to 2.0.0 then you won't get another v2 update suggestion when 2.0.1 comes out

**Reply by gustaff-weldon** [reply]
Perfect, thanks!
